README steps not working

Preface: Am a complete go newbie.

Following the steps in the `README`, the step that executes the tests fails with:
```
# go test ./...
h2quic/client.go:13:2: cannot find package "golang.org/x/net/http2" in any of:
	/usr/local/Cellar/go/1.8.1/libexec/src/golang.org/x/net/http2 (from $GOROOT)
	/Users/lars/go/src/golang.org/x/net/http2 (from $GOPATH)
h2quic/client.go:14:2: cannot find package "golang.org/x/net/http2/hpack" in any of:
	/usr/local/Cellar/go/1.8.1/libexec/src/golang.org/x/net/http2/hpack (from $GOROOT)
	/Users/lars/go/src/golang.org/x/net/http2/hpack (from $GOPATH)
h2quic/client.go:15:2: cannot find package "golang.org/x/net/idna" in any of:
	/usr/local/Cellar/go/1.8.1/libexec/src/golang.org/x/net/idna (from $GOROOT)
	/Users/lars/go/src/golang.org/x/net/idna (from $GOPATH)
h2quic/request_writer.go:13:2: cannot find package "golang.org/x/net/lex/httplex" in any of:
	/usr/local/Cellar/go/1.8.1/libexec/src/golang.org/x/net/lex/httplex (from $GOROOT)
	/Users/lars/go/src/golang.org/x/net/lex/httplex (from $GOPATH)
```

I did do a `go get -t` but those packages did not seem to get installed.

--- a/README.md
+++ b/README.md
@@ -18,9 +18,9 @@ As Google's QUIC versions are expected to converge towards the [IETF QUIC draft]
 
 We currently support Go 1.7+.
 
-Installing deps:
+Installing and updating dependencies:
 
-    go get -t
+    go get -t -u ./...
 
 Running tests:
 

